scripts/publish.js

```diff
@@ -62,6 +62,16 @@ if (!nextVersion) {
   const [version, major, minor, patch] = nextVersion.match(
     /^(\d+)\.(\d+)\.(\d+)$/
   );
+  const currentBranch = shell.exec('git rev-parse --abbrev-ref HEAD', {
+    silent: true
+  }).stdout;
+
+  if (currentBranch.trim() !== 'v' + nextVersion) {
+    console.log(
+      'You must execute this script in the release branch matching SALESFORCEDX_VSCODE_VERSION.'
```

Comment:
How about 'You must execute this script in the release branch matching SALESFORCEDX_VSCODE_VERSION.' --> \`You must execute this script in the release branch matching SALESFORCEDX_VSCODE_VERSION (e.g, release/${SALESFORCEDX_VSCODE_VERSION} or hotfix/${SALESFORCEDX_VSCODE_VERSION}\` and use the template strings feature of JavaScript?